dailymotion-player-objc/DMPlayerViewController.m

```diff
@@ -43,6 +45,7 @@ - (void)setup {
   _duration = NAN;
   _seeking = false;
   _error = nil;
+  _started = false;
```

Comment:
I wasn't sure about this and choose to stay close to the existing code-base. 
Will do.